libs/labelbox/tests/data/export/conftest.py

```diff
@@ -118,18 +118,174 @@ def ontology():
         "color": "#008941",
         "classifications": [],
     }
-    entity_tool = {
+    raster_segmentation_tool = {
```

Comment:
Why was this change, i.e. removed entity_tool and segmentation_tool

Reply:
Oh so regular masks (raster-segmentation) are here I just removed super pixel because it is not only deprecated but completed removed in the app so no point of making ontologies with it anymore. Entity I had to remove this ontology is meant for images and images don't support entity. The old workflow was to just add every single option to an ontology. You cant really do that anymore with regular connect_edit